install_deps.sh

```diff
@@ -73,37 +55,20 @@ then
 	sudo mkdir -p "${opt_dir}"
 	sudo chown $USER: "${opt_dir}"
 
+	if y_or_n "Next: Add apt-get sources list/keys and install necessary packages"; then
+		./scripts/install_apt_deps.sh
+	fi
+
 	if [ ! -d "${temp_dir}/opencv" ] && y_or_n "Next: Install OpenCV from source"; then
-		git clone --branch 3.4.6 https://github.com/opencv/opencv/ "${temp_dir}/opencv"
-		git clone --branch 3.4.6 https://github.com/opencv/opencv_contrib/  "${temp_dir}/opencv_contrib"
-		cmake \
-			-S "${temp_dir}/opencv" \
-			-B "${temp_dir}/opencv/build" \
-			-D CMAKE_BUILD_TYPE=Release \
-			-D CMAKE_INSTALL_PREFIX=/usr/local \
-			-D BUILD_TESTS=OFF \
-			-D BUILD_PERF_TESTS=OFF \
-			-D BUILD_EXAMPLES=OFF \
-			-D BUILD_JAVA=OFF \
-			-D WITH_OPENGL=ON \
-			-D OPENCV_EXTRA_MODULES_PATH="${temp_dir}/opencv_contrib/modules"
-		sudo make -C "${temp_dir}/opencv/build" "-j$(nproc)" install
-		sudo ldconfig -v
+		./scripts/install_opencv.sh
 	fi
 
 	if [ ! -d "${temp_dir}/Vulkan-Headers" ] && y_or_n "Next: Install Vulkan Headers from source"; then
-		git clone https://github.com/KhronosGroup/Vulkan-Headers.git "${temp_dir}/Vulkan-Headers"
-		cmake \
-			-S "${temp_dir}/Vulkan-Headers" \
-			-B "${temp_dir}/Vulkan-Headers/build" \
-			-D CMAKE_INSTALL_PREFIX=install
-		sudo make -C "${temp_dir}/Vulkan-Headers/build" "-j$(nproc)" install
+		./scripts/install_vulkan_headers.sh
 	fi
 
 	if [ ! -d "${opt_dir}/googletest" ] && y_or_n "Next: Install gtest"; then
-		git clone https://github.com/google/googletest --branch release-1.10.0 "${opt_dir}/googletest"
-		cmake -S "${opt_dir}/googletest" -B "${opt_dir}/googletest/build"
-		make -C "${opt_dir}/googletest/build" "-j$(nproc)"
+		./scripts/install_gtest.sh
```

Comment:
How are `opt_dir` and `temp_dir` being passed on to the child scripts?

Reply:
These files now get sourced so the variables get passed through.